### PR TITLE
connecting viz.yaml to siteText.yaml

### PR DESCRIPTION
**Pull request by eread-usgs**
@jiwalker-usgs check this out and see if I followed the pattern correctly.

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -256,24 +256,26 @@ publish:
     id: map-section
     publisher: section
     template: simplefigure
-    depends:
+    depends: siteText
```
Comment (jiwalker-usgs) [inline viz.yaml]: this should be on the next line with something like `text: siteText`  also, siteText will need to be defined in yaml

File: viz.yaml [hunk 2]
```diff
@@ -256,24 +256,26 @@ publish:
...
       map: map-svg
       gages-js: gages-js
     context:
       id: gage-map
       figure: map
-      caption: states
+      caption: siteText.explanationText1
+		  data: ["siteText.title", "siteText.subtitle"]
```
Comment (jiwalker-usgs) [inline viz.yaml]: yaml hates tabs, also we will need the template to handle `data` we can get @mwernimont on that as he is changing the templates anyway

File: viz.yaml [hunk 3]
```diff
@@ -256,24 +256,26 @@ publish:
...
-    depends:
+    depends: siteText
       doyNM_graph: doy-NM
       gages-js: gages-js
     context:
       id: doyNM-graph
       figure: doyNM_graph
-      caption: doyNM
+      caption: siteText.explanationText2
+	  data: siteText.dataSources
```
Comment (jiwalker-usgs) [inline viz.yaml]: again tabs, also I think @ldecicco-USGS is combining this with the section above

**Review by jiwalker-usgs — CHANGES_REQUESTED** [review]
I'll swing by to clarify my comments

**Comment by ldecicco-USGS** [comment]
This good-to-go @jiwalker-usgs ?

**Comment by jiwalker-usgs** [comment]
we'll need to work this out with @mwernimont template changes

**Comment by eread-usgs** [comment]
@jiwalker-usgs check this out, I am not sure I got the formatting on line 282 (adding an intro section for the 2nd figure) quite right.

File: viz.yaml [hunk 4]
```diff
@@ -270,7 +276,9 @@ publish:
     depends:
       header: header
       footer: footer
+	  intro: intro
```
Comment (jiwalker-usgs) [inline viz.yaml]: looks good after tabs removed

File: viz.yaml [hunk 5]
```diff
@@ -270,7 +276,9 @@ publish:
...
       map-section: map-section
+	  intro2: intro
```
Comment (jiwalker-usgs) [inline viz.yaml]: why intro2?

Comment (eread-usgs) [reply viz.yaml]: i dont know, do those need unique IDs? I will call it Intro if not. Its the intro to the 2nd part....

Comment (eread-usgs) [reply viz.yaml]: tabs gone now, let me know if I need to change the intro2 part @jiwalker-usgs

**Review by jiwalker-usgs — APPROVED** [review]

**Comment by eread-usgs** [comment]
should be ready to merge @jiwalker-usgs

**Pull request merged**